docs/tutorials/gluon/logistic_regression_explained.md

```diff
@@ -0,0 +1,215 @@
+    print("Epoch: %s, Training loss: %.2f, Validation loss: %.2f, Validation accuracy: %s" % 
+          (e, cumulative_train_loss, cumulative_val_loss, accuracy.get()[1]))
+
+    # we reset accuracy, so the new epoch's accuracy would be calculate from the blank state
+    accuracy.reset()
+
+```
+
+    Epoch: 0, Training loss: 446.68, Validation loss: 40.19, Validation accuracy: 0.85 <!--notebook-skip-line-->
+
+    Epoch: 1, Training loss: 343.15, Validation loss: 30.82, Validation accuracy: 0.85 <!--notebook-skip-line-->
+    
+    Epoch: 2, Training loss: 187.40, Validation loss: 11.76, Validation accuracy: 0.96 <!--notebook-skip-line-->
+    
+    Epoch: 3, Training loss: 90.18, Validation loss: 10.13, Validation accuracy: 0.98 <!--notebook-skip-line-->
+    
+    Epoch: 4, Training loss: 68.51, Validation loss: 8.69, Validation accuracy: 0.97 <!--notebook-skip-line-->
+    
+    Epoch: 5, Training loss: 67.43, Validation loss: 6.71, Validation accuracy: 0.99 <!--notebook-skip-line-->
+    
+    Epoch: 6, Training loss: 54.76, Validation loss: 7.45, Validation accuracy: 0.98 <!--notebook-skip-line-->
+    
+    Epoch: 7, Training loss: 48.29, Validation loss: 8.56, Validation accuracy: 0.97 <!--notebook-skip-line-->
+    
+    Epoch: 8, Training loss: 50.50, Validation loss: 7.24, Validation accuracy: 0.98 <!--notebook-skip-line-->
+    
+    Epoch: 9, Training loss: 49.42, Validation loss: 7.46, Validation accuracy: 0.97 <!--notebook-skip-line-->
+
+
+## Tip 1: Use only one neuron in the output layer
+
+Despite that there are 2 classes, there should be only one output neuron, because `SigmoidBinaryCrossEntropyLoss` accepts only one feature as an input. 
+
+In case when there are 3 or more classes, one cannot use a single Logistic regression, but should do multiclass regression. The solution would be to increase the number of output neurons to the number of classes and use `SoftmaxCrossEntropyLoss`. 
+
+## Tip 2: Encode classes as 0 and 1
+
+`Sigmoid` function produces values from 0 to 1. `SigmoidBinaryCrossEntropyLoss` uses these values to calculate the   loss by essentially subtracting values and class labels from 1. [Here is the formula](https://mxnet.incubator.apache.org/api/python/gluon/loss.html?highlight=sigmoidbinarycrossentropyloss#mxnet.gluon.loss.SigmoidBinaryCrossEntropyLoss) used for that calculation (we use default version with `from_sigmoid` is False). That's why it is numerically better to have classes encoded in the same range as a `Sigmoid` output with 0 and 1.
+
+If your data comes with a label encoded in a different format, such as -1 and 1, then you can either recode it to 0 and 1 by comparing the initial class to 0, or use another function instead of `Sigmoid`, like [`Tanh`](https://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html?highlight=tanh#mxnet.ndarray.tanh), to produce output in range [-1; 1].
```

Comment:
Not sure if mentioning tanh has any merit here. tanh is just not used for logistic regression. What loss do I use with tanh? I find it confusing.

Reply:
Removed Tanh part.